CONTRIBUTING.MD

```diff
@@ -0,0 +1,103 @@
+# How to contribute
+
+## Making Changes
+
+* Fork the repository on GitHub
+* Create a topic branch from where you want to base your work.
+  * This is usually the master branch.
+  * Only target release branches if you are certain your fix must be on that
+    branch.
+  * To quickly create a topic branch based on master; `git checkout -b
```

Comment:
do we need to introduce our branch names patterns, like fix/... , feat/..., patch/version/... before ?